Automattic/jetpack
# Instant Search: Add an "Expanded" search result format

Fixes #15013.

#### Changes proposed in this Pull Request:
* Adds a new "Expanded" result format.

<img width="732" alt="Screen Shot 2020-06-24 at 11 20 40 AM" src="https://user-images.githubusercontent.com/4044428/85602716-dfdeba00-b60c-11ea-9368-c864d11a35da.png">

* Tweaks the existing "Minimal" result format.

<img width="713" alt="Screen Shot 2020-06-19 at 2 39 46 PM" src="https://user-images.githubusercontent.com/4044428/85178038-bdc6ef80-b23a-11ea-9210-004d989710ee.png">

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Apply this change to your Jetpack installation.
* Try a search. You should still see the previous result format.
* To try the new format, go to the Jetpack Search section of the Customizer and select the "Expanded" result format.

#### Proposed changelog entry for your changes:
* Adds a new, default search result format configurable from the Customizer.
